src/apis/image/apis.ts

```diff
@@ -0,0 +1,5 @@
+import type { PostUploadImagesRes } from './types'
+import { http } from '@utils/http'
+
+export const postUploadImages = (files: FormData) =>
```

Comment:
```suggestion
type PostUploadImagesReq = FormData

export const postUploadImages = (files: PostUploadImagesReq) =>
```

`PostUploadImagesReq`타입으로 선언하는 거 어떠신가요?? `postUploadImages`를 사용하는 쪽에서 Req가 필요할 때 params로 추적하지 않고 타입명으로 추적하는 게 더 빠를 것 같아서 제안드려봅니다!

Reply:
[변경완료](https://github.com/price-offer/offer-fe/pull/179/commits/17abbec909f742c99514d4e2c6f59e8da13e77e8) !